logs/querysample/querysample.go

```diff
@@ -116,3 +133,19 @@ func TransformLogMinDurationStatementToQuerySample(logLine state.LogLine, queryT
 	}
 	return sample, true
 }
+
+func findQueryParameters(paramText string) []null.String {
+	// Handle Query Parameters (available from Postgres 16+)
+	var parameters []null.String
+	// Regular expression to find all values in single quotes or NULL
+	// Query Parameters example: $1 = 'foo', $2 = '123', $3 = NULL, $4 = 'bo''o'
+	re := regexp.MustCompile(`'((?:[^']|'')*)'|NULL`)
+	for _, part := range re.FindAllString(paramText, -1) {
```

Comment:
check if this can be somehow combined with https://github.com/pganalyze/collector/blob/768c8019ab80d9596326b8e976b7371c7f91fc45/logs/querysample/querysample.go#L108

Reply:
well, not really, so I'll keep this as is